src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  static removeFromAlive(animal) {
+    this.alive = this.alive.filter(creature => creature !== animal);
```

Comment:
here you are comparing object to object(reference to reference), either we can't compare their names, because we can have animals with same names.
Let's rename this method to `removeDeadAnimals`, and filter animals which have health > 0

Reply:
Done.